common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/SilentTokenCommandParameters.java

```diff
@@ -43,6 +44,9 @@
 
     private static final Object sLock = new Object();
 
+    @Expose()
+    private final boolean refreshIn;
```

Comment:
Can we pick a name that sounds more like a boolean?

```suggestion
    private final boolean refreshDueToRefreshIn;
```